src/settings.js

```diff
@@ -45,7 +45,7 @@ export default {
     labelPointsConfig: {
         size: 1.5,
         depthTest: true,
-        depthFunc: THREE.NotEqualDepth,
+        depthFunc: THREE.LessEqualDepth,
```

Comment:
Do we need to modify this??

Reply:
For points it should be as it was before, I'll revert